test/e2e/main.go

```diff
@@ -55,23 +63,40 @@ func main() {
 }
 
 func runSuite() error {
-	if err := ExpectOperatorAvailable(); err != nil {
-		glog.Errorf("FAIL: ExpectOperatorAvailable: %v", err)
-		return err
-	}
-	glog.Info("PASS: ExpectOperatorAvailable")
 
-	if err := CreateClusterAutoscaler(); err != nil {
-		glog.Errorf("FAIL: CreateClusterAutoscaler: %v", err)
-		return err
+	expectations := []struct {
+		expect func() error
+		name   string
+	}{
+		{
+			expect: ExpectOperatorAvailable,
+			name:   "[k8s][openshift] Expect operator to be available",
+		},
+		{
+			expect: CreateClusterAutoscaler,
```

Comment:
why is this excluded from [k8s]?

Reply:
Since I am creating different ClusterAutoscaler resources (more constrained one) so the scaling down is quick.

Reply:
what's exactly "I am creating", is there any code somewhere, in the jj may be? couldn't it be created here analogous to [openshift]?

Reply:
Compared to the ClusterAutoscaler for openshift I need following additions to it:

```go
ca.Spec.ResourceLimits.Cores.Min = 1
ca.Spec.ResourceLimits.Memory.Min = 1
ca.Spec.ScaleDown = &autoscalingv1alpha1.ScaleDownConfig{
	Enabled:           true,
	DelayAfterAdd:     "10s",
	DelayAfterDelete:  "10s",
	DelayAfterFailure: "10s",
	UnneededTime:      "10s",
}
```

Reply:
I also explicitly added this in https://github.com/openshift/kubernetes-autoscaler/pull/40/commits/3295fc36c105bf329c9ed90acf621482002585a4#diff-d73fd64bab7fd758991b5a15f551fd02R143

Reply:
I'm confused here, is there anything stopping us from declaring this in code, exactly same we are doing for openshift and just set different param values